couchdb.service

```diff
@@ -10,6 +10,7 @@ User=couchdb
 Group=couchdb
 LimitNOFILE=100000
 KillMode=process
+ExecStopPost=/bin/bash -c 'until /opt/couchdb/erts-10.3/bin/epmd -names | (! grep couchdb); do sleep 1; done'
```

Comment:
`/opt` is maybe too specific?

Reply:
No no it's good to have an absolute path here.

Reply:
Ok thanks!